src/pages/Signup.jsx

```diff
@@ -36,6 +37,13 @@ const SignupScreen = () => {
     setShowPassword(!showPassword);
   };
 
+  const [showCPassword, setCShowPassword] = useState(false);
```

Comment:
try showConfirmPassword and setShowConfirmPassword. Readability is necessary for a clean code base